src/etc/inc/plugins.inc.d/vxlan.inc

```diff
@@ -108,16 +110,18 @@ function vxlan_configure_do($verbose = false, $device = null)
         } else {
             $isChanged = false;
             $current_settings['vxlanid'] = $interfaces_details[$device_name]['vxlan']['vni'];
-            $current_settings['vxlanlocal'] = explode(":", $interfaces_details[$device_name]['vxlan']['local'])[0];
-            $current_settings['vxlanremote'] = explode(":", $interfaces_details[$device_name]['vxlan']['remote'])[0];
+            $current_settings['vxlanlocal'] = $interfaces_details[$device_name]['vxlan']['local'];
+            $current_settings['vxlanlocalport'] = $interfaces_details[$device_name]['vxlan']['localport'];
```

Comment:
@Monviech this looks a bit odd `legacy_interfaces_details()` returns `local` and `remote` as `address:port` combination. Where should the `localport` come from in this case?

Reply:
You are totally right that it is odd. I made some php debug statements and it shows exactly what you pointed out. I have to look more into this.

```
[11-Dec-2023 20:03:24 Etc/UTC] vxlan_configure_do started. Verbose: true, Device: null
[11-Dec-2023 20:03:24 Etc/UTC] Processing device: vxlan1
[11-Dec-2023 20:03:24 Etc/UTC] Before setting current_settings for vxlan1
[11-Dec-2023 20:03:24 Etc/UTC] interfaces_details for vxlan1: Array
(
    [vni] => 50
    [local] => 192.168.3.1:5000
    [group] =>
    [remote] => 192.168.1.2:5000
)
[11-Dec-2023 20:03:24 Etc/UTC] After setting current_settings for vxlan1
[11-Dec-2023 20:03:24 Etc/UTC] Current settings: Array
(
    [vxlanid] => 50
    [vxlanlocal] => 192.168.3.1:5000
    [vxlanlocalport] =>
    [vxlanremote] => 192.168.1.2:5000
    [vxlanremoteport] =>
    [vxlangroup] =>
)
```

Reply:
See my original comment on the explode() use and how the port should be on [1] for local and remote.

Reply:
@fichtner I should have looked into this more thoroughly. Sorry.